setup/instance/inputs.go

```diff
@@ -2,9 +2,7 @@ package instance
 
 import (
 	"fmt"
-
-	"github.com/Sirupsen/logrus"
-	"github.com/quintilesims/layer0/setup/aws"
+	"log"
 )
```

Comment:
The package `github.com/quintilesims/layer0/common/config` needs to be here, otherwise the code will not compile. A reference is made to config.DefaultAWSRegion on line 149, so this package has to be imported.

Reply:
If this is in fact intended behavior, then I would make two separate statements.

1) The help usage isn't clear enough to reflect that the flags are meant as options to exclude an instance type. It's not apparent that `--local=false` is an option looking at the usage help

2) I would argue that the UX for specifying a flag in CLI to exclude something from a listing rather than exclusively listing something is really awkward and non-intuitive.